Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actix-web-opentelemetry"
-version = "0.13.0-alpha.1"
+version = "0.13.0-alpha.2"
```

Comment:
probably don't need to increase this as we haven't shipped the previous alpha

Reply:
Reverted 👍